model.py

```diff
@@ -0,0 +1,111 @@
+class Model:
+	def __init__(self):
+		'''
+		Constructor 
+		'''
+		pass
+
+	def user_info_to_database(nutrition_standard,
```

Comment:
nên đổi naming thành `set_user_info()`, theo convention "getter and setter" á anh.

Reply:
a muốn func này update mấy userinfo to database, ý hiểu của em có phải vậy ko?

Reply:
dạ em cũng hiểu vậy